libs/ui/controls/tagSelector.css

```diff
@@ -184,37 +184,44 @@
   background-color: darkgray;
 }
 
+.tagselect-picker {
+  height: 100%;
+}
+
 .tagselect-modal-cols {
   display: flex;
   height: 100%;
   overflow-x: auto;
 }
 
-.tagselect-modal .col {
+.tagselect-picker .col {
   min-width: 250px;
   margin-right: 20px;
   overflow-y: auto;
   padding-bottom: 10px;
 }
 
-.tagselect-modal .cb {
+.tagselect-picker .cb {
   opacity: 0;
   position: absolute;
   width: 30px;
   height: 30px;
   z-index: 1;
+  cursor: pointer;
 }
 
-.tagselect-modal label {
+.tagselect-picker label {
   position: relative;
-  margin: 2em 0 1em;
-  padding: 0 0 0 40px;
+  padding: 0 10px 0 40px;
   height: 30px;
+  width: 100%;
```

Comment:
I'm seeing an unnecessary horizontal scroll bar when using search, you might want to override the width in `.tagselect-modal-table .search-item label`.